docs/contracts.rst

```diff
@@ -194,9 +194,16 @@ return parameter list for functions.
         function setData(uint a) internal { data = a; }
         uint public data;
     }
+    contract Caller {
+        function readData(){
+            C c = new C();
+            uint local = c.data(); // call accessor
```

Comment:
It is probably better to remove any mention of accessor functions here and then explain the modifiers that are relevant for accessor functions in the section about accessor functions.